### PR TITLE
ERC20 Token Wrapper Test

### PR DESCRIPTION
**Pull request by shuffledex**

### CODE REVIEW

File: src/contract_wrappers/tokens/__tests__/erc20_wrapper.test.ts [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.allowance).once();
+      verify(mockedMethod.callAsync(params.owner, params.spender)).once();
+    });
+  });
+
+  describe('isValidContract', () => {
+    test('should call to isValidContract', async () => {
```
Comment (VictorVicente) [inline src/contract_wrappers/tokens/__tests__/erc20_wrapper.test.ts]: You aren't calling the real method haha.
I realize it when I checked the coverage report. 
Add the call and check if the test is passing.

Comment (shuffledex) [reply src/contract_wrappers/tokens/__tests__/erc20_wrapper.test.ts]: I called each module individually

Comment (VictorVicente) [reply src/contract_wrappers/tokens/__tests__/erc20_wrapper.test.ts]: It is not ok. You are testing `isValidContract` so you need to make a real call to that method here. 
For each of those individual methods must be a different tests. 
This way, we are not covering `isValidContract` method, so if anyone makes a change or delete it, tests won't fail and we won't notice it.

**Pull request merged**